test/Amazon.Lambda.Tools.Test/DeployTest.cs

```diff
@@ -186,6 +186,41 @@ public async Task RunDeploymentNativeAot(bool multipleProjects, bool customMount
             }
         }
 
+        [Fact]
+        public async Task NativeAotNet8WebApp()
```

Comment:
Could you add a comment with the context on why this was added, mentioning that this test ensures we're detecting the inferred `<OutputType>Exe</OutputType>` correctly?

Reply:
Done